src/api/migraphx.py

```diff
@@ -170,6 +170,21 @@ def program(h):
              const=True)
 
 
+@auto_handle
+def onnx_options(h):
+    h.constructor('create')
+    h.method(
+        'add_parameter_shape',
+        api.params(name='const char*', dim_num='size_t', dims='size_t*'),
```

Comment:
This should just be `api.params(name='const char*', dims='std::vector<size_t>')`(although `migraphx::add_parameter_shape` will need to be updated to take a vector). The python will automatically create an additional parameter for the size of dims.